types/context.go

```diff
@@ -45,9 +45,10 @@ type Context struct {
 	// Deprecated: Use HeaderService for hash
 	headerHash []byte
 	// Deprecated: Use HeaderService for chainID and CometService for the rest
-	chainID              string
-	txBytes              []byte
-	logger               log.Logger
+	chainID string
+	txBytes []byte
+	logger  log.Logger
+	// Deprecated: use Cometinfo.GetLastCommit().Votes() instead, will be removed in 0.51
 	voteInfo             []abci.VoteInfo
```

Comment:
nit: this makes it easier to understand that you're referring to a specific field. Alternatively, you can just move the comment to the right of the field.
```suggestion
	logger  log.Logger

	// Deprecated: use Cometinfo.GetLastCommit().Votes() instead, will be removed in 0.51
	voteInfo             []abci.VoteInfo

```